express-post-json/index.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var app = express();
+let nextId = 1;
+const grades = {};
+/*
+grades = {
+    "course": "JavaScript",
+    "id": 4,
+    "name": "Yuliia",
+    "score": 2000000
+}
+*/
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const newGradesArr = [];
+  for (const key in grades) {
+    newGradesArr.push(grades[key]);
+  }
+  // newGradesArr.push(grades);
+  res.json(newGradesArr);
+});
+
+app.post('/api/grades', (req, res) => {
+  const gradesObj = req.body; // given obj ???
+  gradesObj.id = nextId;
+  grades[gradesObj.id] = gradesObj;
+  nextId++;
+  res.send(gradesObj);
+  res.status(201);
```

Comment:
If you set the status code after sending the response, it will not have any effect. Instead, set the status code before `res.send()`.